functions/triggers-getting-started/go/handler.go

```diff
@@ -0,0 +1,48 @@
+package handler
+
+import (
+	"fmt"
+	"io"
+	"math/big"
+	"net/http"
+	"strconv"
+)
+
+func factorial(n int) *big.Int {
+	var f big.Int
+	f.MulRange(1, int64(n))
+	return &f
+}
+
+// This handle function comes frome our examples and is not modified at all.
+func Handle(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodPost {
+		// SQS triggers are sent as POST requests.
+		http.Error(w, "Method not allowed", http.StatusMethodNotAllowed)
+		return
+	}
+
+	// The SQS trigger sends the message content in the body.
+	body, err := io.ReadAll(r.Body)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	n, err := strconv.Atoi(string(body))
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	result := factorial(n)
+	fmt.Printf("go: factorial of %d is %s\n", n, result.String())
```

Comment:
Yeah it's [here](https://github.com/scaleway/serverless-examples/pull/46/files#diff-b6ca9978fadb7ce860048cc0ac4530a0fbd54971cb80874535387d8251f0304aR41) ^^

Do you prefer `io.WriteString` or `w.Write` ?

Reply:
That's fine like this 👌

Reply:
I kept it because it's required for the tests for go, python & node, and I wanted to have the same behaviors across runtimes. As we discussed, I added a comment explaining that this body is not used by the trigger.

Also added a comment about the status code, as the status is used to decide if it should be retried.